.github/workflows/pipeline.yml

```
@@ -37,21 +38,23 @@ jobs:
         with:
           useLockFile: false
 
+      - name: Check Types
+        run: npm run type-check
+
       - name: Lint code
         run: npm run lint
 
-      # TODO: reenable on v4 + run tsc
-    #      - name: Check format
-    #        run: npm run format:check
+      - name: Check format
+        run: npm run format:check
 
   tests:
     name: Tests (Node v${{ matrix.node }} - ESLint v${{ matrix.eslint }})
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node: [ '10.12', '10', '12.0', '12', '14' ]
-        eslint: [ '5', '6', '7' ]
+        node: [ 10.12, 10, 12, 14 ]
```

Add `Node@12.0` to check against the minimum 12 requirement

```suggestion
        node: [10.12, 10, 12.0, 12, 14]
```

---

I don't see why we need node 12.0 and node 12.X (we are only adding 10.12 because it's the very minimum one for the plugin)

---

We have 2 very minimum ones: 10.12 & 12.0

---

fair enough